src/instruments/config.py

```diff
@@ -139,16 +130,11 @@ def load_instruments(conf_file_name, conf_path="/"):
         such processing will occur independently of the value of ``conf_path``.
     """
 
-    if yaml is None:
-        raise ImportError(
-            "Could not import ruamel.yaml, which is required " "for this function."
-        )
-
```

Comment:
`ruamel.yaml` is not an optional dependency. So I see no need for this check... unless we make it an optional dependency ;)